### PR TITLE
Add HiGenEvent object to heavy-ion miniAOD event content

### PR DESCRIPTION
**Pull request by mandrenguyen**
#### PR description:

The HIGenEvent object stores a few gen-level variables that are relevant for heavy ion collisions.  It's stored in AOD, we would like to keep it for miniAOD, too.  The size of the object is negligible 

#### PR validation:

Tested w/ wf 159

### CODE REVIEW

**Comment by jpata** [comment]
@cmsbuild please test

**Comment by jpata** [comment]
For posterity, can you post some more detailed information about the size?

**Comment by jpata** [comment]
assign @cms-sw/xpog-l2

File: PhysicsTools/PatAlgos/python/slimming/MicroEventContent_cff.py [hunk 1]
```diff
@@ -148,6 +148,7 @@
     'keep *_hiEvtPlane_*_*',
     'keep *_hiEvtPlaneFlat_*_*',
     'keep QIE10DataFrameHcalDataFrameContainer_hcalDigis_ZDC_*',
+    'keep edmGenHIEvent_heavyIon_*_*',
```
[the remark this answers is not in the document]
Comment (mandrenguyen) [reply PhysicsTools/PatAlgos/python/slimming/MicroEventContent_cff.py]: Yep.  Sorry, fixed now.

**Comment by mandrenguyen** [comment]
@jpata It's just a few floats.  3 kb per event if I understand the output of edmDumpEventSize correctly.  It's near the bottom in the list of collections.  Should be negligible.

**Comment by clacaputo** [comment]
@cmsbuild please test

**Comment by clacaputo** [comment]
> @jpata It's just a few floats, I think. 3 kb per event if I understand the output of edmDumpEventSize correctly. It's near the bottom in the list of collections. Should be negligible.

Hi @mandrenguyen , on how many events have you performed this measurement?

**Comment by clacaputo** [comment]
Hello @mandrenguyen , any update on this https://github.com/cms-sw/cmssw/pull/36961#issuecomment-1048808247?
Thanks

**Comment by mandrenguyen** [comment]
@clacaputo The added size is negligible.  I suggest just running wf 159 to check it.

**Comment by mandrenguyen** [comment]
The ratio for 1 event in 159 is 2104127./2103802. = 1.0001545
wf 159 runs peripheral events so lighter than the average PbPb event.  In our normal embedded MC this extra collection will be even more negligible.

**Comment by clacaputo** [comment]
+reconstruction

- no reco changes expected or observed
- the event size increase measured is negligible https://github.com/cms-sw/cmssw/pull/36961#issuecomment-1056760067

**Comment by mariadalfonso** [comment]
+xpog

**Comment by qliphy** [comment]
+1

**Pull request merged**